docs/application/native/guides/app-management/cion.md

```diff
@@ -0,0 +1,235 @@
+
+    ```xml
+    <privileges>
+       <privilege>http://tizen.org/privilege/d2d.datasharing</privilege>
+       <privilege>http://tizen.org/privilege/internet</privilege>
+    </privileges>
+    ```
+
+2.  To use the functions of the Cion API, include the `<cion.h>` header file in your application:
+
+    ```cpp
+    #include <cion.h>
+    ```
+
+<a name="server_client"></a>
+## Communicate in server-client style
+
+Cion API provides functions to communicate with the other applications in server-client style.
+
+Cion client can try to discover the server to communicate. If server is discovered, client can try to connect to the server.
+After client is connected to server, client can send data and paylaod.
+
+Cion client can be used as follows:
+
+1. Create the client handle:
+
+    ```cpp
+    int ret;
+    cion_client_h client;
+
+	ret = cion_client_create(&client, "TEST_SERVICE", NULL);
+    ```
+
+2. Discover the server to connect:
+
+    ```cpp
+    static void __client_server_discovered_cb(const char *service_name,
+		const cion_peer_info_h peer_info, void *user_data)
+    {
+    }
+    ```
+    ```cpp
+    ret = cion_client_try_discovery(client, __client_server_discovered_cb, NULL);
+    ```
+
+3. Connect to the server:
+
+    ```cpp
+    ret = cion_client_connect(client, peer_info);
+    ```
+
+    > [!NOTE]
+    > You can get the `peer_info` from discovered callback function.
+
+    > [!NOTE]
+    > You can add callback function using `cion_client_add_connection_result_cb` to get the connection result.
+
+4. Send the data:
+
+    ```cpp
+    unsigned char str[] = "test_data";
+    unsigned char *return_data = NULL,
+    unsigned int return_size;
+
+    ret = cion_client_send_data(client, strlen((const char*)str), 5000, &return_data, &return_size);
+    ```
+5. Send the payload:
+
+    ```cpp
+    static void __cion_client_payload_async_result_cb(
+        const cion_payload_async_result_h result, void *user_data);
+    ```
+    ```cpp
+    cion_payload_h payload;
+    unsigned char data[] = "test_payload";
+
+    ret = cion_payload_create(&payload, CION_PAYLOAD_TYPE_DATA);
+    ret = cion_payload_set_data(payload, data, sizeof(data));
+
+    ret = cion_client_send_payload_async(client, payload, __cion_client_payload_async_result_cb, NULL);
+    ```
+
+Cion server can listen the request from the client. If a connection request comes from the client, the server can accept connection.
+After connection is accepted, the server can receive data and payload.
+
+Cion server can be used as follows:
+
+1. Create the server handle:
+
+    ```cpp
+    cion_server_h server;
+
+    ret = cion_server_create(&server, "TEST_SERVICE", "MY_DISPLAY_NAME", NULL);
+    ```
+
+2. Server is ready to listen the requests from the client:
```

Comment:
Line 117: An ordered (numbered) list should provide a step or instruction to the reader. I'd recommend rephrasing the sentence. I'm adding a suggestion below:
```suggestion
2. Make sure the server is ready to listen to the requests from the client:
```

Reply:
sentence is changed